client/extensions/woocommerce/app/settings/email/email-settings/components/internal-notification.js

```diff
@@ -61,14 +61,14 @@ const InternalNotification = ( {
 			<ListItemField className="components__notification-component-input">
 				<FormTextInput
 					className={ isPlaceholder ? 'components__is-placeholder' : null }
-					isError={ emailValidationError }
+					isError={ checked && emailValidationError }
 					name={ item.field }
 					onChange={ change }
 					value={ recipient }
-					placeholder={ placeholder }
+					placeholder={ isPlaceholder ? '' : translate( 'Recipient email address.' ) }
```

Comment:
It looks like `isPlaceholder` contains loading status which shouldn't affect setting a string here. `placeholder={ translate( 'Recipient email address.' ) }` should be fine.

Optional: Also maybe rename `isPlaceholder` to `isLoading` or something, just to prevent any confusion?